makefile.gen

```diff
@@ -94,94 +115,104 @@ Debug: debug
 Release: release
 Asm: asm
 
-.PHONY: clean
-
 cleantmp:
 	$(RM) -f $(RES_RS)
+.PHONY: cleantmp
 
 cleandep:
 	$(RM) -f $(DEPS)
+.PHONY: cleandep
 
 cleanlst:
 	$(RM) -f $(LSTS)
+.PHONY: cleanlst
 
 cleanres: cleantmp
 	$(RM) -f $(RES_H) $(RES_DEP) $(RES_DEPS)
+.PHONY: cleanres
 
 cleanobj:
-	$(RM) -f $(OBJS) out/sega.o out/rom_head.bin out/rom_head.o out/rom.out
+	$(RM) -f $(OBJS) $(OUT)/sega.o $(OUT)/rom_head.bin $(OUT)/rom_head.o $(OUT)/rom.out
+.PHONY: cleanobj
 
 clean: cleanobj cleanres cleanlst cleandep
-	$(RM) -f out.lst out/cmd_ out/symbol.txt out/rom.nm out/rom.wch out/rom.bin
+	$(RM) -f out.lst $(OUT)/cmd_ $(OUT)/symbol.txt $(OUT)/rom.nm $(OUT)/rom.wch $(OUT)/rom.bin

```

Comment:
I don't know what out.lst is exactly, but it looks like it belongs in $(OUT), so I'm moving it.

Reply:
Done.